### PR TITLE
Seeds

### PR DESCRIPTION
**Pull request by katrinagh**
Making the seeds, je verifie apres

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,132 @@
...
+
+estimated_delivery_times =["10-15min", "15-20min", "10-20min"]
+
+delivery_fees =["2chf", "4chf", "5chf", "7chf"]
+
+puts "Creating new pharmacies"
+    Store.create!(
+    title: titles[rand(1..8)],
+    description:  descriptions[rand(1..8)],
+    image: images[rand(1..8)],
```
Comment (SoniaBisinger) [inline db/seeds.rb:39]: image_url

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,132 @@
...
+    rating: ratings[rand(1..5)],
```
Comment (SoniaBisinger) [inline db/seeds.rb:40]: rand(1..5)

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,132 @@
...
+    estimated_delivery_time: estimated_delivery_tims[rand(1..3)],
```
Comment (SoniaBisinger) [inline db/seeds.rb:41]: attention -> times

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,132 @@
...
+    delivery_fee: delivery_fees[rand(1..4)]
```
Comment (SoniaBisinger) [inline db/seeds.rb:42]: delivery_fees: delivery_fees selon table

File: db/seeds.rb [hunk 5]
```diff
@@ -5,3 +5,132 @@
...
+    )
+
+
+
+  Article.create!(
+  description: "cpr 500 mg blister 16 pce",
+  title: "Dafalgan",
+  weight: "10g",
+  category: "Pain",
+  price: "CHF 2.50"
```
Comment (SoniaBisinger) [inline db/seeds.rb:52]: faire integer

File: db/seeds.rb [hunk 6]
```diff
@@ -5,3 +5,132 @@
...
+    estimated_delivery_time: estimated_delivery_tims[rand(1..3)],
+    delivery_fee: delivery_fees[rand(1..4)]
+    )
+
+
+
+  Article.create!(
+  description: "cpr 500 mg blister 16 pce",
+  title: "Dafalgan",
+  weight: "10g",
```
Comment (SoniaBisinger) [inline db/seeds.rb:50]: integer

**Review by PilcodeurDev — APPROVED** [review]

**Review by SoniaBisinger — APPROVED** [review]

**Pull request merged**